pydm/utilities/stylesheet.py

```diff
@@ -26,6 +28,12 @@ def clear_cache():
     __style_data = None
 
 
+def merge_widget_stylesheet(widget, stylesheet_file_path=None):
+    curr_style = widget.styleSheet() or ""
+    env_style = _get_style_data(stylesheet_file_path) or ""
+    widget.setStyleSheet(env_style + curr_style)
```

Comment:
I wonder how Qt handles conflicting styles on the same selector? Does the last one win? We might need a note that the ordering matters in $PYDM_STYLESHEET.

Reply:
If conflicts are found, the last defined rule is applied. I tested it here before coming up with the PR. Also, styles at the widget will always win over styles defined at parent.
I think I can do a better job highlighting the fact that order matters at the documentation. The same concept as LD_LIBRARY_PATH or any other PATH will apply there.

Reply:
@mattgibbs I added a new section to the documentation to talk about `Customizing the Look and Feel` and covering how conflicts are sorted by Qt Style Sheet engine. Would you please review it again so we can merge it in?